### PR TITLE
increase the resource limit for the graph namespace

### PR DESCRIPTION
**Pull request by harshad16**
increase the resource limit for the graph namespace
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

Require some cushion in resource quota for newer deployments. 

a-graph

### CODE REVIEW

**Review by tumido — CHANGES_REQUESTED** [review]
Why do you want to change requests to be larger than limits? I don't understand.

File: cluster-scope/base/core/namespaces/thoth-graph-prod/resourcequota.yaml [hunk 1]
```diff
@@ -5,8 +5,8 @@ metadata:
 spec:
   hard:
     limits.cpu: '12'
```
Comment (4n4nd) [inline cluster-scope/base/core/namespaces/thoth-graph-prod/resourcequota.yaml:7]: when you increase `requests.cpu`, `limits.cpu` needs to increase as well. Requested resources cannot be more than the resource limits.

Comment (harshad16) [reply cluster-scope/base/core/namespaces/thoth-graph-prod/resourcequota.yaml:7]: very good point, it was mistake, fixing it :sweat_smile:

**Comment by harshad16** [comment]
> Why do you want to change requests to be larger than limits? I don't understand.

Just to check if the reviewers are paying attention or not :sweat_smile: 
jokes aside: it was an honest mistake, need to check my eyes.


Ready for review , cc: @4n4nd @tumido

**Review by 4n4nd — COMMENTED** [review]
/lgtm

**Review by HumairAK — APPROVED** [review]
/lgtm
/approve

**Pull request merged**